core/client/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -463,9 +461,9 @@ public void initialize(URI uri, org.apache.hadoop.conf.Configuration conf) throw
     FileStatus[] ret = new FileStatus[statuses.size()];
     for (int k = 0; k < statuses.size(); k++) {
       URIStatus status = statuses.get(k);
-      // TODO(hy): Replicate 3 with the number of disk replications.
-      ret[k] = new FileStatus(status.getLength(), status.isFolder(), 3, status.getBlockSizeBytes(),
-          status.getCreationTimeMs(), status.getCreationTimeMs(), null, null, null,
+      ret[k] = new FileStatus(status.getLength(), status.isFolder(), BLOCK_REPLICATION_CONSTANT,
```

Comment:
@calvinjia , do you know why we always return 3 as the replication level? Will this confuse YARN/Spark scheduler?